File: Components/RewriteMe.Domain/Messages/LanguageVersion.cs
using System;
using RewriteMe.Domain.Enums;

namespace RewriteMe.Domain.Messages
{
    public class LanguageVersion
    {
        public Guid Id { get; set; }

        public Guid InformationMessageId { get; set; }

        public string Title { get; set; }

        public string Message { get; set; }

        public string Description { get; set; }

        public Language Language { get; set; }

        public bool SentOnOsx { get; set; }

        public bool SentOnAndroid { get; set; }
    }
}

File: WebControlPanelClient/src/app/_models/language-version.ts
import { Language } from '../_enums/language';

export class LanguageVersion {
    id: string;
    informationMessageId: string;
    title: string;
    message: string;
    description: string;
    language: Language;
    sentOnOsx: boolean;
    sentOnAndroid: boolean;
}
